protocols/1577/custom_chip_PCR_prep.ot2.py

```diff
@@ -0,0 +1,84 @@
+from opentrons import labware, instruments
+
+metadata = {
+    'protocolName': 'Custom Chip PCR Preparation',
+    'author': 'Nick <protocols@opentrons.com>',
+    'source': 'Custom Protocol Request'
+}
+
+# create custom labware
+chip_name = '3x8-chip'
+if chip_name not in labware.list():
+    labware.create(
+        chip_name,
+        grid=(8, 3),
+        spacing=(9, 7.75),
+        diameter=5,
+        depth=0,
+        volume=20
+    )
+
+strips_name = 'Tempassure-PCR-strips-200ul'
+if strips_name not in labware.list():
+    labware.create(
+        strips_name,
+        grid=(12, 8),
+        spacing=(9, 9),
+        diameter=6,
+        depth=20,
+        volume=200
+    )
+
+# load labware
+tubes = labware.load('opentrons-tuberack-2ml-eppendorf', '1', 'reagent tubes')
+chip = labware.load(chip_name, '2')
+strips = labware.load(strips_name,
+                      '3',
+                      'strips')
+tips10 = labware.load('tiprack-10ul', '4')
+tips50 = labware.load('opentrons-tiprack-300ul', '5')
+
+# pipettes
+p10 = instruments.P10_Single(mount='right', tip_racks=[tips10])
+p50 = instruments.P50_Single(mount='left', tip_racks=[tips50])
+
+# reagents
+h2o = tubes.wells('A1')
+red_buffer = tubes.wells('B1')
+primer = tubes.wells('C1')
+dna_pol = tubes.wells('D1')
+mix_tube = tubes.wells('A2')
+
+# create master mix, enough for 4 extra samples
+p50.transfer(15.5*28, h2o, mix_tube.top(), blow_out=True)
+p50.transfer(5*28, red_buffer, mix_tube.top(), blow_out=True)
+p50.transfer(1*28, primer, mix_tube.top(), blow_out=True)
+p50.pick_up_tip()
+p50.transfer(0.25*28, dna_pol.bottom(), mix_tube, new_tip='never')
```

Comment:
1. Double check that they are cool with increasing mm volumes
2. Add in variable for num samples
3. Add in variable for flow rate